app/views/spree/admin/users/_form.html.haml

```diff
@@ -12,6 +12,9 @@
             = check_box_tag "user[spree_role_ids][]", role.id, @user.spree_roles.include?(role), id: "user_spree_role_#{role.name}"
             = label_tag role.name
       = hidden_field_tag "user[spree_role_ids][]", ""
+    = f.field_container :locale do
+      = f.label :locale, t(".locale")
+      = f.select :locale, locales_available, class: "fullwidth"
```

Comment:
I think that this is good in the form. There may be a case that someone accidentally switches languages and is then unable to switch back. At least an admin can do it for them then.